packages/aws-rfdk/lib/deadline/test/version-query.test.ts

```diff
@@ -57,44 +55,6 @@ test('VersionQuery constructor full version', () => {
     ],
   }));
   expectCDK(stack).to(haveResourceLike('AWS::Lambda::Function', {
-    Code: {
-      S3Bucket: {
-        Ref: VERSION_QUERY_ASSET.Bucket,
-      },
-      S3Key: {
```

Comment:
This block has been causing problems due to the S3 bucket name and object key hashes always changing. I know Eugene has a PR open already where he was going to remove this section, so when my tests failed due to the hash changing I decided to just delete the `Code` section rather than fix the hashes.

Reply:
Can we obtain these values using `stack.resolve(asset.s3ObjectKey)` and `stack.resolve(asset.bucket.buckwetName)`?

Reply:
Unless I'm missing something, if we are pulling values from the stack and then using those values in the expectation, we're essentially saying, "Make sure that the code in the stack is the code that we found in the stack."

Reply:
This is fair - I don't see us losing much by removing this.